src/xdocs/checks/annotation/suppresswarnings.xml

```diff
@@ -163,50 +173,64 @@ class TestA {
   @SuppressWarnings(true?&quot;all&quot;:&quot;unused&quot;) // ok
   void foo4(int param) {}
 }
-@SuppressWarnings(&quot;unchecked&quot;) // ok
-class TestB {}
+// ok below as The warning 'unchecked' is a valid warning type
+@SuppressWarnings(&quot;unchecked&quot;)
+class Test1 {}
         </source>
 
-        <p>
+        <p id="Example2-config">
           To configure the check so that the &quot;unchecked&quot; and &quot;unused&quot;
           warnings cannot be suppressed on anything
           but variable and parameter declarations.
         </p>
         <source>
-&lt;module name=&quot;SuppressWarnings&quot;&gt;
-  &lt;property name=&quot;format&quot;
-      value=&quot;^unchecked$|^unused$&quot;/&gt;
-  &lt;property name=&quot;tokens&quot;
-    value=&quot;
-    CLASS_DEF,INTERFACE_DEF,ENUM_DEF,
-    ANNOTATION_DEF,ANNOTATION_FIELD_DEF,
-    ENUM_CONSTANT_DEF,METHOD_DEF,CTOR_DEF
-    &quot;/&gt;
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;SuppressWarnings&quot;&gt;
+      &lt;property name=&quot;format&quot;
+          value=&quot;^unchecked$|^unused$&quot;/&gt;
+      &lt;property name=&quot;tokens&quot;
+        value=&quot;
+        CLASS_DEF,INTERFACE_DEF,ENUM_DEF,
+        ANNOTATION_DEF,ANNOTATION_FIELD_DEF,
+        ENUM_CONSTANT_DEF,METHOD_DEF,CTOR_DEF
+        &quot;/&gt;
+    &lt;/module&gt;
+  &lt;/module&gt;
 &lt;/module&gt;
         </source>
-
-        <p>Example:</p>
+        <p id="Example2-code">Example:</p>
         <source>
-@SuppressWarnings(&quot;&quot;) // ok
-class TestA {
-  @SuppressWarnings(&quot;&quot;) // ok
+// ok below as applied to a CLASS_DEF, allowed by the tokens property
+@SuppressWarnings(&quot;&quot;)
```

Comment:
@relentless-pursuit please take some time to play around with this check, I think we have a misunderstanding of how it works: https://checkstyle.sourceforge.io/checks/annotation/suppresswarnings.html#SuppressWarnings

I would expect this comment to be like:
```suggestion
// ok below, since we are only checking for '^unchecked$|^unused$'
@SuppressWarnings(&quot;&quot;)
```

If a token is configured in the `tokens` property, these are exactly the tokens that we are checking. Please review all comments again.

Reply:
done. please let me know your feedback on the same.